cli/tests/test_commit_template.rs

```diff
@@ -120,6 +120,19 @@ fn test_log_author_timestamp_utc() {
     @  2001-02-02 21:05:07.000 +00:00
     ◉  1970-01-01 00:00:00.000 +00:00
     "###);
+
+    test_env.add_env_var("TZ", "Asia/Kolkata");
```

Comment:
Maybe better to use absolute value. I don't know if DST exists in these areas, but tzdata may be updated in future.

Reply:
I checked that they don't have DST, but absolute offset is still better (I didn't realize you could do that). I've done that now, but the offset is confusingly with the opposite sign compared to what I would expect.

I've also disabled the test on Windows because it doesn't seem to respect the `TZ` environment variable.